movieApp.js

```diff
@@ -0,0 +1,423 @@
+      "Ralf Moeller",
+      "Spencer Treat Clark",
+    ],
+    poster_url:
+      "https://www.themoviedb.org/t/p/w600_and_h900_bestv2/ty8TGRuvJLPUmAR1H1nRIsgwvim.jpg",
+    price: "115.00",
+  },
+  {
+    id: 12,
+    title: "The Silence of the Lambs",
+    description:
+      "A young F.B.I. cadet must receive the help of an incarcerated and manipulative cannibal killer to help catch another serial killer, a madman who skins his victims.",
+    movie_year: 1991,
+    director: "Jonathan Demme",
+    actors: [
+      "Jodie Foster",
+      "Anthony Hopkins",
+      "Lawrence A. Bonney",
+      "Kasi Lemmons",
+      "Lawrence T. Wrentz",
+      "Scott Glenn",
+      "Anthony Heald",
+      "Frankie Faison",
+      "Don Brockett",
+    ],
+    poster_url:
+      "https://www.themoviedb.org/t/p/w600_and_h900_bestv2/rplLJ2hPcOQmkFhTqUte0MkEaO2.jpg",
+    price: "85.00",
+  },
+  {
+    id: 13,
+    title: "Schindler's List",
+    description:
+      "In German-occupied Poland during World War II, industrialist Oskar Schindler gradually becomes concerned for his Jewish workforce after witnessing their persecution by the Nazis.",
+    movie_year: 1993,
+    director: "Steven Spielberg",
+    actors: [
+      "Liam Neeson",
+      "Ben Kingsley",
+      "Ralph Fiennes",
+      "Caroline Goodall",
+      "Jonathan Sagall",
+      "Embeth Davidtz",
+      "Malgorzata Gebel",
+      "Shmuel Levy",
+      "Mark Ivanir",
+    ],
+    poster_url:
+      "https://www.themoviedb.org/t/p/w600_and_h900_bestv2/c8Ass7acuOe4za6DhSattE359gr.jpg",
+    price: "120.00",
+  },
+  {
+    id: 14,
+    title: "Se7en",
+    description:
+      "Two detectives, a rookie and a veteran, hunt a serial killer who uses the seven deadly sins as his motives.",
+    movie_year: 1995,
+    director: "David Fincher",
+    actors: [
+      "Brad Pitt",
+      "Morgan Freeman",
+      "Gwyneth Paltrow",
+      "Kevin Spacey",
+      "R. Lee Ermey",
+      "John C. McGinley",
+      "Julie Araskog",
+      "Mark Boone Junior",
+      "John Cassini",
+    ],
+    poster_url:
+      "https://www.themoviedb.org/t/p/w600_and_h900_bestv2/69Sns8WoET6CfaYlIkHbla4l7nC.jpg",
+    price: "95.00",
+  },
+];
+
+const nowStreaming = document.querySelector(".now-streaming");
+
+// const img = document.createElement("img");
+// const readMoreBtn = document.createElement("button");
+
+for (let movie of movies) {
+  //Creat a cotainer to store the Image and readMore Button
+
+  const movieContainer = document.createElement("div");
+  movieContainer.className = "movie-container";
+
+  //
+  //  Create an img element and set its src attribute
+  const img = document.createElement("img");
+  img.width = 200;
+  img.src = movie.poster_url;
+
+  // Create a button element
+  const readMoreBtn = document.createElement("button");
+  readMoreBtn.textContent = "Read More";
+  readMoreBtn.className = "accordion";
+  readMoreBtn.style.backgroundColor = "teal";
+  readMoreBtn.style.marginTop = "10px";
```

Comment:
I was trying to create this function, but my other function get stuck with this. I would like to discuss this in next class how to do that. mean while I merge this branch to do the other homework of last week and after doing that will try again. if it will still persist with the same problem, I will need your help.